File: 6th Semester/Programming Paradigms/NP2 Exercises - Part 2/ex07.py
# Escreva um programa que exiba a tabuada do número digitado, onde o usuário possa escolher o inicio
# e o fim da tabuada.

tabuada = int(input("Digite o numero que se deseja mostrar a tabuada: "))
inicio = int(input("Digite o inicio da tabuada: "))
fim = int(input("Digite o fim da tabuada: "))
for i in range(inicio, fim+1):
    print(f"{tabuada} x {i} = {tabuada*i}")
    

File: 6th Semester/Programming Paradigms/NP2 Exercises - Part 2/ex07.c
// Escreva um programa que exiba a tabuada do número digitado, onde o usuário possa escolher o inicio
// e o fim da tabuada.

#include <stdio.h>

int main()
{
    int inicio, fim, tabuada;
    printf("Digite o numero que se deseja mostrar a tabuada: ");
    scanf("%d", &tabuada);
    printf("Digite o inicio da tabuada: ");
    scanf("%d", &inicio);
    printf("Digite o fim da tabuada: ");
    scanf("%d", &fim);
    for (int i = inicio; i <= fim; i++)
        printf("%d x %d = %d\n", tabuada, i, i * tabuada);
    return 0;
}